Makefile

```diff
@@ -186,8 +208,10 @@ docker-generate-crds:
 	docker run --rm -v $(PWD):$(WORKDIR) -v $(GOPATH)/pkg/mod:/go/pkg/mod:delegated \
 		-v $(shell go env GOCACHE):/root/.cache/go-build:delegated --env GO111MODULE=on \
 		--env https_proxy=$(https_proxy) --env http_proxy=$(http_proxy) \
-		$(BUILD_IMAGE):$(OPERATOR_SDK_VERSION) /bin/bash -c 'operator-sdk generate crds'
+		orangeopensource/casskop-build:v0.19.4 /bin/bash -c 'controller-gen crd paths=./pkg/apis/... output:dir=./deploy/crds schemapatch:manifests=./deploy/crds'
```

Comment:
@AKamyshnikova can you change it back ? The version was updated. Maybe you changed that for your tests?

Reply:
Oops, sorry this fix this.